### ISSUE TITLE
Unable to upload Microsoft Excel sheet in Slack Channel

### ISSUE THREAD

**Issue by RamkumarManavalan** [microsoft/BotFramework-Composer#9072]
Details of the Composer version I use:
- Version:  2.1.1
- Electron: 8.2.4
- Chrome: 80.0.3987.165
- NodeJS: 12.13.0
- V8: 8.0.426.27-electron.0

I use the "File or Attachment" option to accept files from users in Slack. Every other common file types work, except for Microsoft Excel/Word files. The "File or Attachment" does not return any result back, but shows a preview of the file properly. Please see the picture below.
<img width="521" alt="slack-file-attachment-error" src="https://user-images.githubusercontent.com/7835046/160161596-87a7dff3-2288-4dbf-8baa-bd6fd8c8820f.png">

Any help in resolving this?  Thanks.

**Comment by stevkan** [comment]
@RamkumarManavalan, at the moment I am unable to repro the issue. To be clear, are you using the Slack adapter package, found in the Package Manager, in your Composer bot? Also, what is the size of the Excel document that you are trying to send?

**Comment by RamkumarManavalan** [comment]
No, I dont use slack adapter. I created a slack app and used its credentials to configure in the bot's slack channel configuration in azure portal.

The size is of KBs (less tha 500 KB).

**Comment by stevkan** [comment]
@RamkumarManavalan, apologies to you as it seems I misread your initial post. Based on your description, I am now able to repro the issue. Looking at the activity with an attached Excel file, it appears that the `activity.attachments` property is not populating with the files `contentType`, `contentUrl`, and `name` properties. As you can see in the attached image, the activity with an attached JSON file is structured correctly whereas the Excel attachment is not:

![image](https://user-images.githubusercontent.com/18403946/160909057-6491c44b-0043-48af-9889-e7c699940979.png)

![image](https://user-images.githubusercontent.com/18403946/160909064-ec636fc6-fb4b-4a1e-a27d-20bb74a24f51.png)

I am able to repro the issue in both a C# Composer bot as well as in a code-first JS bot. The activity, including the `attachments` property, is constructed in the Slack channel connector as the incoming Slack message arrives. This bug will need to be addressed by our Services team.

**@Services** team, can someone please investigate this further?
